managed-ledger/src/test/java/org/apache/bookkeeper/mledger/impl/NonDurableCursorTest.java

```diff
@@ -824,5 +824,16 @@ void deleteNonDurableCursorWithName() throws Exception {
         assertEquals(Iterables.size(ledger.getCursors()), 0);
     }
 
+    @Test
+    public void testMessagesConsumedCounterInitializedCorrect() throws Exception {
+        ManagedLedger ledger = factory.open("testMessagesConsumedCounterInitializedCorrect",
```

Comment:
`ledger` should be closed finaly

Reply:
Good suggestion. Already fixed

Reply:
Sorry, There is no need to close it explicitly. `MockedBookKeeperTestCase#tearDown` will close all ledgers in `factory.shutdownAsync()`. LGTM